fastlane/Fastfile

```diff
@@ -212,6 +212,27 @@ platform :ios do
     scan(scheme: "StoreKitTests", derived_data_path: "scan_derived_data")
   end
 
+  desc "Update swift package commit"
+  lane :update_swift_package_commit do
```

Comment:
this used to live directly in the Fastfile for the `SPMIntegration` integration test project, but I moved it here since it's usable by more targets.